doing-business-with-us/index.html

```diff
@@ -18,38 +18,37 @@
     <h1>Doing Business with Us</h1>
     <p class="h3">
         We work with vendors who can help us find or develop the right tools, people, strategies,
-        infrastructure, and programs in support of our mission. We do this work in the open.
-        Here you’ll find information about both existing and future procurements. Any questions?
+        infrastructure, and programs in support of our mission. We do this work in the open. Here
+        you'll find information about both existing and future procurements. Any questions?
         <a href="mailto:david.gragan@cfpb.gov">Contact our head of procurement.</a>
     </p>
 
     <section class="block block__bg block__border-top block__border-bottom block__flush-sides">
-        <h1 class="h2">Get started</h1>
+        <h1 class="h2">Opportunities to work with us</h1>
         <div class="content-l content-l__main">
             <div class="content-l_col content-l_col-1-3">
                 <h2 class="h4">
                     <span class="step-slug char-icon char-icon__1">Step one</span>
-                    Register with System of Award Management
+                    Register with System of Award Management (SAM)
                 </h2>
                 <p class="short-desc">
-                    Processes for procurement and awards across the federal government
-                    are managed in the System of Award Management (SAM).
-                    Anyone interested in federal procurements or awards should
-                    first register with SAM.
+                    Processes for procurement and awards across the federal government are managed
+                    in the System of Award Management. Anyone interested in federal procurements or
```

Comment:
Nope. See [our written style guide (GHE link)](https://ghe.domain/flapjack/written-style-guide/blob/master/new-style-guide-structure.md#3-terms). Can't link directly there, but "federal" is in that Terms section.